packages/manager/modules/hub/src/components/carousel/carousel.html

```diff
@@ -0,0 +1,44 @@
+<div class="carousel">
+    <div class="carousel-inner">
+        <div
+            data-ng-repeat="notification in $ctrl.notifications track by $index"
```

Comment:
use id instead of $index

Reply:
Notification id is a `"string"` so using index